STAC-extensions.yaml

```diff
@@ -531,6 +531,11 @@ components:
         maxItems: 6
```

Comment:
This is strange, there's still the old min/maxItems in the file, in additon to the new oneOf. But it should not. Does that get included from the OAFeat part? Should be fixed...

Reply:
Ugh, I thought this would be an easy fix. Good catch on the OAFeat part. I updated it but forgot to run the generation stuff. So I did, and now have:

```
        type: array
        oneOf:
          - minItems: 4
            maxItems: 4
          - minItems: 6
            maxItems: 6
          - minItems: 4
            maxItems: 4
          - minItems: 6
            maxItems: 6
```

Not helpful. Going to try to dig into how merging works... Any ideas are welcome.

Reply:
Ok, ended up just removing most of the BBOX elements from STAC.yaml, as it says we extend the OAFeat one, but then we replicate a number of fields. So made it so it was just the description, where we add an example. 

Hopefully everything is right now.